testing_and_setup/compass/ocean/coastal/USDEQU120cr10rr2/build_mesh/config_base_mesh.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<config case="base_mesh">
+
+	<add_executable source="mesh_converter" dest="MpasMeshConverter.x"/>
+	<add_executable source="cell_culler"    dest="MpasCellCuller.x"/>
+
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py"                dest="build_mesh.py"/>
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/jigsaw_driver.m"              dest="jigsaw_driver.m"/>
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/triangle_jigsaw_to_netcdf.py" dest="triangle_jigsaw_to_netcdf.py"/>
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/mesh_definition_tools.py"     dest="mesh_definition_tools.py"/>
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/coastal_tools.py"             dest="coastal_tools.py"/>
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/inject_bathymetry.py"         dest="inject_bathymetry.py"/>
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/inject_meshDensity.py"        dest="inject_meshDensity.py"/>
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/inject_preserve_floodplain.py" dest="inject_preserve_floodplain.py"/>
+	<add_link source_path="script_test_dir"     source="define_base_mesh.py"                                                         dest="define_base_mesh.py"/>
+	<add_link source_path="jigsaw-geo-matlab"   source="."                                                                           dest="jigsaw-geo-matlab"/>
+        <add_link source_path="bathymetry_database" source="SRTM15_plus/earth_relief_15s.nc"                                             dest="earth_relief_15s.nc"/>
```

Comment:
I think we are missing a step to the `paraview_vtk_field_extractor.py` here...

Reply:
Thanks for pointing this out. I resolved this in the latest commit.

Reply:
I think there should be a flag to optionally run the paraview extraction step. Arguably, it's not even useful to have it at this stage since the mesh hasn't been culled yet.

Reply:
I completely agree.